distributed/worker.py

```diff
@@ -129,11 +129,18 @@ def __init__(self, scheduler_ip, scheduler_port, ip=None, ncores=None,
         self.memory_limit = memory_limit
         if memory_limit:
             try:
-                from zict import Buffer, File, Func
+                from zict import Buffer, File, Func, LMDB, Sieve
             except ImportError:
                 raise ImportError("Please `pip install zict` for spill-to-disk workers")
             path = os.path.join(self.local_dir, 'storage')
-            storage = Func(dumps_to_disk, loads_from_disk, File(path))
+            file_storage = Func(dumps_to_disk, loads_from_disk, File(path))
+            lmdb_path = os.path.join(self.local_dir, 'lmdb')
+            lmdb = Func(dumps_to_disk, loads_from_disk, LMDB(lmdb_path))
```

Comment:
I'm mostly being cautious here, in case LMDB ends up slowish for very large values on some systems (Windows? special filesystems? who knows :-)).